tools/test.py

```diff
@@ -21,6 +21,7 @@ def parse_args():
     parser.add_argument(
         '--eval',
         default='mAP',
+        nargs='+',
```

Comment:
why multiple args?

Reply:
In hand pose estimation, we will need multiple metrics, such as PCK, AUC...
We can choose to use different metrics.

Reply:
then why change metric to metrics?
doesn't that mean we want the user to specify one metric?